<launch>
  <?ignore  
    <node name="gps_decoder" pkg="slam" type="gps_decoder.py" output="screen"/>
  ?>
  <node name="gps_decoder" pkg="slam" type="gps_decoder.py" output="screen"/>
  <node name="imu_decoder" pkg="slam" type="imu_decoder"/>
  <node name="kalman_fusion" pkg="slam" type="kalman_fusion"/>
  <node name="kalman_fusion_visualizer" pkg="slam" type="kalman_fusion_visualizer.py" output="screen"/>
</launch>
